Gems/Multiplayer/Code/Source/Debug/MultiplayerConnectionViewportMessageSystemComponent.cpp

```diff
@@ -45,11 +45,13 @@ namespace Multiplayer
     {
         AZ::RPI::ViewportContextNotificationBus::Handler::BusConnect(
             AZ::RPI::ViewportContextRequests::Get()->GetDefaultViewportContextName());
+        MultiplayerEditorServerNotificationBus::Handler::BusConnect();
     }
 
     void MultiplayerConnectionViewportMessageSystemComponent::Deactivate()
     {
         AZ::RPI::ViewportContextNotificationBus::Handler::BusDisconnect();
+        MultiplayerEditorServerNotificationBus::Handler::BusDisconnect();
```

Comment:
NP: We usually disconnect in reverse order from Connect to ensure any potential dependencies are honored. This probably isn't an issue here but it'd be good to maintain that order.

Reply:
i like it! fixed :)